src/Reflector.php

```diff
@@ -39,63 +39,344 @@
  */
 final class Reflector
 {
+    /**
+     * TODO: Migrate to Roave/BetterReflection once it supports constants
+     */
     private const KNOWN_INTERNAL_CONSTANTS = [
```

Comment:
@sebastianbergmann is there any constants from extensions PHPUnit relies on which are not in this list?

Reply:
Could be, don't know. Don't have a tool handy that lists all constants used.

Reply:
I'll try to have a look before merging that one. As long as the constants are not supported by BetterReflection it's a very brittle system so it would be nice it works fine at least for PHPUnit

Reply:
Outdated thanks to @kukulich we can leverage BetterReflector for which constants support has been added